src/qt/forms/debugwindow.ui

```diff
@@ -694,16 +703,6 @@
              </property>
             </widget>
            </item>
-           <item>
-            <widget class="QPushButton" name="btnClearTrafficGraph">
```

Comment:
If we're deleting this, maybe there should be a way for the user to insert a reference line?

For now, I'd move removal of anything to a separate PR.

Reply:
The user can effectively remove the data by deleting the .dat file - is there any basis for being able to do this while the client is running?